docker-compose.local.yml

```diff
@@ -12,7 +12,7 @@ services:
     # NOTE: If you need to access the database from an external tool,
     # expose this port and restart the container.
     # ports:
-    #   - 5432:5432
+      - 5432:5432
```

Comment:
Undo. You shouldn't need to expose the ports to your machine.

Reply:
Done